caffe2/CMakeLists.txt

```diff
@@ -374,14 +374,14 @@ if(USE_ROCM)
   # Call again since Caffe2_HIP_INCLUDES is extended with ATen include dirs.
   hip_include_directories(${Caffe2_HIP_INCLUDES})
 
-  filter_list(__caffe2_hip_srcs_cpp Caffe2_HIP_SRCS "\\.(cc|cpp|cu)$")
+  filter_list(__caffe2_hip_srcs_cpp Caffe2_HIP_SRCS "\\.(cu|hip)$")
```

Comment:
If we have changed the hip filename extension, we shouldn't match for cu anymore, right?

Reply:
Deserves a comment